graylog2-web-interface/src/components/common/EntityFilters/CreateFilterDropdown.tsx

```diff
@@ -79,7 +79,7 @@ const CreateFilterDropdown = ({ filterableAttributes, filterValueRenderers, onCr
                              buttonTitle="Create Filter"
                              onToggle={onToggleDropdown}
                              closeOnSelect={false}
-                             dropdownZIndex={1000}>
+                             dropdownZIndex={1050}>
```

Comment:
This is unrelated to the purpose of this PR, but avoids the dropdown being hidden when used on e.g. drawers.